src/components/homesections/homesections.js

```diff
@@ -365,63 +369,23 @@ import ServerConnections from '../ServerConnections';
         imageLoader.lazyChildren(elem);
     }
 
-    function getContinueWatchingFetchFn(serverId) {
-        return function () {
-            const apiClient = ServerConnections.getApiClient(serverId);
-            const screenWidth = dom.getWindowSize().innerWidth;
-
-            let limit;
-            if (enableScrollX()) {
-                limit = 12;
-            } else {
-                limit = screenWidth >= 1920 ? 8 : (screenWidth >= 1600 ? 8 : (screenWidth >= 1200 ? 9 : 6));
-                limit = Math.min(limit, 5);
-            }
-
-            const options = {
-                Limit: limit,
-                Recursive: true,
-                Fields: 'PrimaryImageAspectRatio,BasicSyncInfo',
-                ImageTypeLimit: 1,
-                EnableImageTypes: 'Primary,Backdrop,Thumb',
-                EnableTotalRecordCount: false,
-                MediaTypes: 'Video'
-            };
-
-            return apiClient.getResumableItems(apiClient.getCurrentUserId(), options);
-        };
-    }
-
-    function getContinueWatchingItemsHtml(items) {
-        const cardLayout = false;
-        return cardBuilder.getCardsHtml({
-            items: items,
-            preferThumb: true,
-            shape: getThumbShape(),
-            overlayText: false,
-            showTitle: true,
-            showParentTitle: true,
-            lazy: true,
-            showDetailsMenu: true,
-            overlayPlayButton: true,
-            context: 'home',
-            centerText: !cardLayout,
-            allowBottomPadding: false,
-            cardLayout: cardLayout,
-            showYear: true,
-            lines: 2
-        });
-    }
-
-    function loadResumeVideo(elem, apiClient) {
+    function loadResume(elem, apiClient, headerText, mediaType) {
```

Comment:
```suggestion
    const dataMonitorHints = {
        'Audio': 'audioplayback,markplayed',
        'Video': 'videoplayback,markplayed'
    };

    function loadResume(elem, apiClient, headerText, mediaType) {
```

_Use `Add suggestion to batch` if you want a single commit._